src/librawspeed/metadata/ColorFilterArray.h

```diff
@@ -27,16 +27,15 @@
 namespace RawSpeed {
 
 enum CFAColor {
-  CFA_COLOR_MIN = 0,
+  // see also DngDecoder
   CFA_RED = 0,
   CFA_GREEN = 1,
   CFA_BLUE = 2,
-  CFA_GREEN2 = 3,
+  CFA_INVALID = 3, // keep the unit tests happy
```

Comment:
And when the dcraw-style filter pattern is parsed, they only signify which cell has which uniquie 'color' id, not which *specific* color (r/g/b) the cell has